controllers/apply/awards-for-all/lib/contacts.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const get = require('lodash/fp/get');
+const { safeHtml } = require('common-tags');
+
+function getContactFullName(data, type) {
+    const fieldName =
```

Comment:
Rather than varying on the type could we pass the `seniorContactName` or `mainContactName` data in directly?